sfm/message_consumer/sfm_ui_consumer.py

```diff
@@ -0,0 +1,36 @@
+# from __future__ import absolute_import
+import logging
+import sys
+# from django.conf import settings
+from sfmutils.consumer import BaseConsumer
+from ui.models import Harvest, SeedSet
+
+log = logging.getLogger(__name__)
+
+STATUS_SUCCESS = "completed success"
```

Comment:
These could be imported from https://github.com/gwu-libraries/sfm-utils/blob/master/sfmutils/harvester.py.